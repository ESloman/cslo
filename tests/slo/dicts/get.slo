var map = {"a": 1, "b": 2, "c": 3};
println(map.get("a"));     # 1
println(map.get("d"));     # nil
println(map.get("d", 4));  # 4
