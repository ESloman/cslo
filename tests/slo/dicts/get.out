1
nil
4
